src/views/open-api-docs-preview-v2/server.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// Utilities
+import getPropsFromPreviewData from './utils/get-props-from-preview-data'
+// Constants
+import { COOKIE_ID, API_ROUTE, DYNAMIC_PARAM } from './constants'
+// Types
+import type { GetServerSidePropsContext, GetServerSidePropsResult } from 'next'
+import type { OpenApiDocsViewV2Props } from 'views/open-api-docs-view-v2/types'
+import type { OpenApiPreviewV2InputValues } from './components/open-api-preview-inputs'
+import { OpenApiDocsPreviewV2Props } from '.'
+
+/**
+ * We expect the `HASHI_ENV` environment variable to be set to `production`
+ * only if we're actually in production.
+ */
+const IS_PRODUCTION = process.env.HASHI_ENV === 'production'
```

Comment:
Since the page is available in production (the current open api docs preview), can this be changed to `process.env.VERCEL_ENV === 'production'`?
```suggestion
const IS_PRODUCTION = process.env.VERCEL_ENV === 'production'
```

Reply:
Yeah great point! TBH I'm not really sure why we have `HASHI_ENV`... maybe related to testing production-ish things locally or something? 🤔 Does seem worth switching to use `VERCEL_ENV`, feels more standard.